demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -388,12 +397,50 @@ def field_mapping(self):
             "default_data_source_id": "defaultDataSource",
         }
 
-    # TODO - implementing all those 3 methods
-    def validate_structure(self) -> List[StructureError]:
-        return []
-
     def raw_data(self) -> dict:
         raise NotImplementedError
 
+    @property
     def strict_object(self):
-        raise NotImplementedError
+        raise NotImplementedError("This object has a different behavior")
+
+    def validate_structure(self) -> List[StructureError]:
+        """
+        This method uses the parsed data and attempts to build a Pydantic (strict) object from it.
+        Whenever the data and schema mismatch, we store the error using the 'structure_errors' attribute,
+        which will be read during the ST110 validation run.
+        In Pack, we need to check two files: the metadata and the RNs json files, so we override the
+        method for combing all the pydantic errors from the both files.
+        """
+        pydantic_error_list: List[StructureError] = []
+
+        # validate Rn's files
+        for file in self.path.glob("ReleaseNotes/*.json"):
+            validate_structure(FileType.RELEASE_NOTES_CONFIG, file, pydantic_error_list)
+
+        # validate pack metadata file
+        validate_structure(
+            FileType.METADATA,
+            Path(self.path, PACK_METADATA_FILENAME),
+            pydantic_error_list,
+        )
+
+        return pydantic_error_list
+
+
+def validate_structure(
+    file_type: FileType, file: Path, pydantic_error_list: list
```

Comment:
instead of passing and checking `file_type`, change the check to be `if file.stem == "pack_metadata"`